test/parallel/test-tls-over-http-tunnel.js

```diff
@@ -24,6 +24,9 @@ const common = require('../common');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 
+// This test ensures that the data throught tls over http tunnel
```

Comment:
Typo: `throught` -> `through`

Also, probably `data received through` rather than just `data through`?

Reply:
changes made. Thank you @Trott for feedback